sdk/core/Azure.Core/src/Shared/AzureCoreSharedEventSource.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.Tracing;
+
+namespace Azure.Core.Diagnostics
+{
+    [EventSource(Name = EventSourceName)]
+    internal sealed class AzureCoreSharedEventSource : EventSource
```

Comment:
Event sources can't be shared. You can't create more than 1 instance with the same name.

Reply:
Do you have a suggestion for how to deal with the eventSource reference in shared source?

Reply:
Avoid is all I can say. ES don't work with the shared source.

Reply:
I just removed it as logging these events can be avoided in the short term.

Reply:
You could use partial classes. Require that projects that use this define a partial that declares the attribute. Maybe you could enforce that they (project maintainers) at least know it's required by declaring a partial method here that must be implemented in the class they create, along with a comment that explains what else (declare the attribute) they must do.

Just a thought.